web/src/features/hfiCalculator/components/CalculatedPlanningAreaCells.tsx

```diff
@@ -60,9 +89,10 @@ const CalculatedPlanningAreaCells = (props: CalculatedCellsProps) => {
             ></MeanIntensityGroupRollup>
             <FireStartsCell areaName={props.areaName} />
             <PrepLevelCell
-              meanPrepLevel={false}
+              meanPrepLevelBoolean={false}
```

Comment:
what is `meanPrepLevelBoolean` used for? Is there a clearer name that could be used instead?

Reply:
That's what is being used to tell the cell if it is calculating a daily prep level or the average for the prep period. Maybe averagePrepLevel?